exasol_script_languages_container_ci_setup/main.py

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python3
+#
+from exasol_script_languages_container_ci_setup.cli.cli import cli
+# noinspection PyUnresolvedReferences
```

Comment:
you can either do the imports in the `__init__.py` or by adding them a list, this avoids the noinspection thing, which caused trouble in the past

Reply:
Done